package/yast2-security.changes

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Mon Mar  7 09:51:41 UTC 2016 - knut.anderssen@suse.com
+
+- Removing the need for a diferent branch for different display
+  manager behaviour. (bnc#946889).
```

Comment:
This looks like a message for ourselves, not something user-oriented. The real feature is that it can handle both types of DM